influxql/engine.go

```diff
@@ -130,11 +144,8 @@ func (p *Planner) planExpr(e *Executor, expr Expr) (Processor, error) {
 
 // planCall generates a processor for a function call.
 func (p *Planner) planRawQuery(e *Executor, v *VarRef) (Processor, error) {
-	// Convert the statement to a simplified substatement for the single field.
-	stmt, err := e.stmt.Substatement(v)
-	if err != nil {
-		return nil, err
-	}
+	stmt := e.stmt
```

Comment:
To be clear, because you're only support 1 field, I presume this is why `Substatement` is not being called.